### ISSUE TITLE
A few suggestions

### ISSUE THREAD

**Issue by Traik** [Automattic/Iris#17]
From the most important to me to the least important:
1) Having alpha or rgba() would make it complete.
2) I'm not sure if this is possible or not but absolute positioning of color picker would be a cool improvement (so that it can float above elements like in Photoshop).
3) Maybe no message "pick a color" at all? It is quite clear anyway. I would add an alternative mode where upon initialization it still shows old input field and to the right of that input field you see a solid little square - exactly like this person did: http://dribbble.com/shots/232627-Custom-colour-picker (it's probably Iris).
4) Additional controls (at least as an option) where user can manually put R, G, B or H, L, S numbers is sometimes useful to me in Photoshop as well.

Basically, I think that companies like Adobe or Autodesk deliver nice colorpickers in their products that can be used for further improvements. And for a person that is half-wordpress-user-half-designer (like a person setting up unified brand color for entire WordPress website through admin panel) some more options like number (4) might be useful.

I hope that some of my ideas will help make it better :)
